CHANGES.md

```diff
@@ -1,5 +1,50 @@
 # Changes
 
+## edge-23.1.1
+
+This edge release introduces a number of different fixes changes to the proxy.
+The proxy has been updated to initialize routes lazily, which means service
+profile routes will now only show up in the metrics when a route is used. In
+the extensions, old (`ServerAuthorization`) resources have been converted to
+`AuthorizationPolicy` -- as part of this change, redundant policy resources
+have been cleaned up. A bug in the destination controller that could
+potentially lead to stale pods being considered in the load balancer has been
+fixed; operations that could previously result in this behavior are now
+infallible. Support has been added for `Pod Security Admission`, used instead
+of `Pod Security Policy`, as part of this change, some of the extension charts
+have been modified to include a `cniEnabled` flag that will impact the policy
+used. 
```

Comment:
PSA is the recommended alternative to PSP, but we still support the latter (although disabled by default). And sorry I didn't update the PR description, but we finally didn't have to add the `cniEnabled` flag.

Reply:
ooo that makes sense, I'll update.